db/migrate/20160628233655_create_comments.rb

```diff
@@ -0,0 +1,9 @@
+class CreateComments < ActiveRecord::Migration
+  def change
+    create_table :comments do |t|
+      t.text     :body
+      t.integer  :comment_id 
```

Comment:
Same issue as with your posts migration -- Rails will automatically assign an id to a comment.

Since comments belong to posts, however, you do need to create a column for `post_id` so that we can attach a comment to a particular post.